lab_2_keywords_cooccurrence/main.py

```diff
@@ -123,8 +202,32 @@ def extract_candidate_keyword_phrases_with_adjoining(candidate_keyword_phrases:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
-
+    if not (check_list(candidate_keyword_phrases, tuple, False) and check_list(phrases, str, False)):
+        return None
+    possible_pairs = {}
+    for i in range(len(candidate_keyword_phrases) - 1):
+        pair = candidate_keyword_phrases[i], candidate_keyword_phrases[i + 1]
+        if pair not in possible_pairs:
+            possible_pairs[pair] = 1
+        else:
+            possible_pairs[pair] += 1
+    appropriate_pairs = []
+    for pair in possible_pairs:
+        if possible_pairs.get(pair, 0) > 1:
```

Comment:
here you do not need get as you iterate over dictionary keys and they are 100% present in the dictionary

Reply:
done